src/main/java/ru/sbtqa/tag/pagefactory/api/I18nApi.java

```diff
@@ -0,0 +1,15 @@
+package ru.sbtqa.tag.pagefactory.api;
+
+import ru.sbtqa.tag.pagefactory.Page;
+import ru.sbtqa.tag.qautils.i18n.I18N;
+
+import java.util.Locale;
+
+public class I18nApi {
```

Comment:
Why didn't you put it into I18N class?

Reply:
because I18N doesn't have i18n resource bundle